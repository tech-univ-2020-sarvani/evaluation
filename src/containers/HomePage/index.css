.container{
    display: flex;
    justify-content: center;
    align-items: center;
    flex-direction: column;
    align-content: center;
    padding-top: 200px;
    font-family: 'Poppins';
}