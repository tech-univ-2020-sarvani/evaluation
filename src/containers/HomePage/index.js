/* eslint-disable react/require-default-props */
import React from 'react';
import './index.css';
import PropTypes from 'prop-types';
import Button from '../../components/Button';
import Nav from '../../components/Nav';

function HomePage(props) {
  const { testId } = props;
  return (
    <div className="homePage" data-testid={testId}>
      <Nav />
      <div className="container">
        <p>:((</p>
        <br />
        <h3>seems a bit empty in here..</h3>
        <Button path="/genres">sync</Button>
      </div>
    </div>
  );
}
HomePage.propTypes = {
  testId: PropTypes.string,
};

export default HomePage;
