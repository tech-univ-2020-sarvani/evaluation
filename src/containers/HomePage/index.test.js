import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import HomePage from './index';
import './index.css';

describe('the HomePage container', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<HomePage testId="test-homepage" />);

    expect(asFragment()).toMatchSnapshot();
  });
});
