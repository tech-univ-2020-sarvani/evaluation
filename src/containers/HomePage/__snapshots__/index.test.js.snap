// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the HomePage container should render correctly 1`] = `
<DocumentFragment>
  <div
    class="homePage"
    data-testid="test-homepage"
  >
    <div
      class="NavBar"
    >
      <p>
        <b>
          Record
        </b>
        Shelf
      </p>
    </div>
    <div
      class="container"
    >
      <p>
        :((
      </p>
      <br />
      <h3>
        seems a bit empty in here..
      </h3>
      <div
        class="buttonContainer"
      >
        <button
          class="button"
          type="button"
        >
          sync
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
